### PR TITLE
Updated UVA Faculty

### PR DESCRIPTION
**Pull request by westre3**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -35,7 +35,6 @@ Aamir Cheema,Monash University,http://www.aamircheema.com,chWbG70AAAAJ
 Aanjhan Ranganathan,Northeastern University,https://www.aanjhan.com,Jid_EAkAAAAJ
 Aaram Yun,UNIST,http://aaramyun.unist.ac.kr,NOSCHOLARPAGE
 Aaron Bernstein,Rutgers University,http://bernsteinaaron.com,NOSCHOLARPAGE
-Aaron Bloomfield,University of Virginia,https://engineering.virginia.edu/faculty/aaron-bloomfield,NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv:38]: Please explain / justify this removal. He still appears here: https://engineering.virginia.edu/departments/computer-science/faculty.

File: csrankings.csv [hunk 2]
```diff
@@ -2409,6 +2408,7 @@ Christopher A. Leckie,University of Melbourne,http://findanexpert.unimelb.edu.au
 Christopher Allen,University of Kansas,http://www.eecs.ku.edu/people/faculty/callen,pFEURKIAAAAJ
 Christopher Amato,Northeastern University,http://www.ccs.neu.edu/home/camato,-8-sD-sAAAAJ
 Christopher Bailey-Kellogg,Dartmouth College,http://www.cs.dartmouth.edu/~cbk,NOSCHOLARPAGE
+Christopher Barrett,University of Virginia,https://engineering.virginia.edu/faculty/christopher-l-barrett/NOSCHOLARPAGE
```
Comment (emeryberger) [inline csrankings.csv]: Ill-formed -- you need a comma before NOSCHOLARPAGE.

File: csrankings.csv [hunk 3]
```diff
@@ -8707,6 +8705,7 @@ Matthew D. Egbert,University of Auckland,https://unidirectory.auckland.ac.nz/peo
 Matthew D. Green,Johns Hopkins University,https://isi.jhu.edu/~mgreen,X0XWAGkAAAAJ
 Matthew D. Sinclair,University of Wisconsin - Madison,http://pages.cs.wisc.edu/~sinclair,gaDqxaQAAAAJ
 Matthew Doolan,Australian National University,https://cecs.anu.edu.au/people/matt-doolan,w4INCC8AAAAJ
+Matthew Dwyer,University of Virginia,http://matthewbdwyer.github.io/ZRKCcEAAAAJ&hl
```
Comment (emeryberger) [inline csrankings.csv]: Ill-formed; need a comma after the home page.

Comment (emeryberger) [reply csrankings.csv]: This is also not the correct DBLP name: https://dblp.uni-trier.de/pers/hd/d/Dwyer:Matthew_B=

PLEASE check all the boxes in the commit contribution guidelines (and do what it says).

File: csrankings.csv [hunk 4]
```diff
@@ -11949,6 +11949,7 @@ Sean W. Smith,Dartmouth College,http://www.cs.dartmouth.edu/~sws,hjaMuwsAAAAJ
 Sean Warnick,Brigham Young University,https://cs.byu.edu/faculty/sean,k7WBrxoAAAAJ
 Sebastian Angel,University of Pennsylvania,https://cis.upenn.edu/~sga001,pNaNj8EAAAAJ
 Sebastian Boring,University of Copenhagen,http://www.diku.dk/english/staff/?pure=en/persons/431157,Y_QMLKAAAAAJ
+Sebastian Elbaum,University of Virginia,https://engineering.virginia.edu/faculty/sebastian-elbaum/swPW5FYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Another ill-formed entry.

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by westre3** [comment]
* Aaron Bloomfield was removed because he is not tenured/tenure-track. Currently, the web page you linked to does not differentiate between non-tenure-track, teaching-track faculty and tenured/tenure-track faculty.

* The entry for Matthew Dwyer that was ill-formed has been removed because it had been accidentally added as a duplicate.

* The other ill-formed entries have been corrected.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Thanks!

**Pull request merged**